pkg/isb/interfaces.go

```diff
@@ -51,6 +51,7 @@ type BufferWriter interface {
 	BufferWriterInformation
 	io.Closer
 	Write(context.Context, []Message) ([]Offset, []error)
+	IsFull() bool
```

Comment:
Do wee need to make it part of the interface? this means all the writers need to implement it, but it's not used anywhere else.

Reply:
Addressed.